FPWD/2020-06-18/index.html

```diff
@@ -1981,6 +1981,24 @@ <h2 id="x6-did-methods"><bdi class="secno">6. </bdi>DID Methods<a class="self-li
             </td>
           </tr>
 
+        <tr>
```

Comment:
I'm pretty sure the published FPWD is not supposed to be updated.

Reply:
Correct, it shouldn't be.